configure

```diff
@@ -297,7 +297,7 @@ if [ $ocaml_major -ge 5 ]; then
     # If findlib has been configured -sitelib $(ocamlc -where) then there's
```

Comment:
The comment above mentions `META` files but with this patch no `META` files are looked at so I think the comment should be updated to avoid confusion.

(Apologies for the misplaced comment, but Github does not allow me to comment on a part of code that's outside of the patch context unfortunately)

Reply:
The comment is still correct - OCaml 5.0+ installs its own META files under the stdlib directory! The rest of the test is working out whether it is necessary to add `$ocaml_core_stdlib` to `$ocamlpath` in order to access those `META` files.